lib/workers/repository/index.ts

```diff
@@ -59,7 +59,7 @@ export async function renovateRepository(
         }
         logger.debug(`Automerged but already retried once`);
       } else {
-        await ensureDependencyDashboard(config, branches);
+        await ensureDependencyDashboard(config, branches, packageFiles);
```

Comment:
don't the packageFiles exist in the config? check it out they might xD config is a big object

Reply:
as I remember that I checked if packageFiles exists in config object but it does not.